### ISSUE TITLE
Standard Deviation not calculated Correctly

### ISSUE THREAD

**Issue by jack-champagne** [iputils/iputils#332]
Using ping, the statistic mdev given on completion is inaccurate as an estimate for standard deviation.

As an example:
```
 ping dlford.io
PING dlford.io (174.126.87.71) 56(84) bytes of data.
64 bytes from 174-126-87-71.cpe.sparklight.net (174.126.87.71): icmp_seq=1 ttl=54 time=81.7 ms
64 bytes from 174-126-87-71.cpe.sparklight.net (174.126.87.71): icmp_seq=2 ttl=54 time=90.9 ms
64 bytes from 174-126-87-71.cpe.sparklight.net (174.126.87.71): icmp_seq=3 ttl=54 time=81.3 ms
64 bytes from 174-126-87-71.cpe.sparklight.net (174.126.87.71): icmp_seq=4 ttl=54 time=82.8 ms
64 bytes from 174-126-87-71.cpe.sparklight.net (174.126.87.71): icmp_seq=5 ttl=54 time=83.8 ms
^C
--- dlford.io ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4002ms
rtt min/avg/max/mdev = 81.274/84.104/90.949/3.537 ms
```

The standard deviation for this sample of pings should actually be closer to 3.92 ms (if I were to for example put in the travel times into excel and get the std. dev.)

**Comment by paul-demo** [comment]
The difference appears to just be the difference between the "sample" and the "population" definition. See Wikipedia for an explanation.

**Comment by jack-champagne** [comment]
Yes, I saw in the manual docs that it was population standard deviation but I thought that treating it as a sample would be better for estimating the true variance of the population. If I wanted to predict ranges for future deviations in ping time, applying bessel's correction first is more accurate.

However, if this is out of scope and unimportant to what this library aims to deliver, we can close this issue.

**Comment by nmeyerhans** [comment]
On Tue, May 25, 2021 at 09:02:58AM -0700, Jack Champagne wrote:
>    Yes, I saw in the manual docs that it was population standard deviation
>    but I thought that treating it as a sample would be better for estimating
>    the true variance of the population. If I wanted to predict ranges for
>    future deviations in ping time, applying bessel's correction first is more
>    accurate.
> 
>    However, if this is out of scope and unimportant to what this library aims
>    to deliver, we can close this issue.

I'd be very hesitant to change the meaning of the existing fields, since
they're well established at this point.  However, collecting and/or
reporting additional statistics might be interesting.  It might be
preferable to report the additional information only if explicitly
requested at runtime (via a flag, etc), but I haven't thought too deeply
about it.

noah
